.github/workflows/quarkus.yml

```
@@ -11,67 +11,68 @@ on:
   - cron: '0 3 * * *'
 
 env:
-  # Workaround testsuite locale issue
-  LANG: en_US.UTF-8
-  DB_USER: hibernate_orm_test
-  DB_PASSWORD: hibernate_orm_test
-  DB_NAME: hibernate_orm_test
   COMMON_MAVEN_ARGS: "-e -B --settings .github/mvn-settings.xml --fail-at-end"
-  NATIVE_TEST_MAVEN_ARGS: "-Dtest-containers -Dstart-containers -Dquarkus.native.native-image-xmx=5g -Dnative -Dnative.surefire.skip -Dformat.skip -Dno-descriptor-tests install -DskipDocs"
+  DB_NAME: hibernate_orm_test
+  DB_PASSWORD: hibernate_orm_test
+  DB_USER: hibernate_orm_test
+  GRAALVM_HOME: ${{ github.workspace }}/graalvm
+  JAVA_HOME: ${{ github.workspace }}/jdk
+  LANG: en_US.UTF-8 # Workaround testsuite locale issue
   MX_GIT_CACHE: refcache
-  MX_PYTHON: python3.9
   MX_PATH: ${{ github.workspace }}/mx
-  JAVA_HOME: ${{ github.workspace }}/jdk
+  MX_PYTHON: python3.9
```

```suggestion
  MX_PYTHON: python3.8
```

It's a good change to make this consistent with the main workflow and in accordance to https://github.com/graalvm/mx/issues/249#issuecomment-1069344236.